test/parser/caps_test.exs

```diff
@@ -17,19 +17,26 @@ defmodule Membrane.H264.CapsTest do
   end
 
   @video_parameters %{
-    "10-720p" => {:high, 1280, 720},
-    "100-240p" => {:high, 320, 240},
-    "20-360p-I422" => {:high_4_2_2, 480, 360},
-    "10-720p-main" => {:main, 1280, 720},
-    "10-720p-no-b-frames" => {:high, 1280, 720},
-    "100-240p-no-b-frames" => {:high, 320, 240}
+    "10-720p" => {:high, 1280, 720, {30, 1}},
+    "100-240p" => {:high, 320, 240, {30, 1}},
```

Comment:
Do we have a resolution that was failing before here?

Reply:
done